### PR TITLE
[Workers] Adds tutorial on sending emails with Resend

### PR DESCRIPTION
**Pull request by megaconfidence**
This PR adds a new tutorial illustrating how to send an email from Workers using [Resend](https://resend.com/)

### CODE REVIEW

File: content/workers/tutorials/send-emails-with-resend/index.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+	},
+};
+
+```
+
+## Add your domain to Resend
+
+If you don’t already have a Resend account, you can sign up for a [free account here](https://resend.com/signup). After signing up, go to `Domains` using the side menu, and click the button to add a new domain. On the modal, enter the domain you want to add and then select a region.
+
+Next, you’re presented with a list of DNS records to add to your Cloudflare domain. On your Cloudflare dashboard, select the domain you entered earlier and navigate to `DNS` > `Records`. Copy/paste the DNS records ( i.e DKIM, SPF, and DMARC records) from Resend to your Cloudflare domain.
```
Comment (kodster28) [inline content/workers/tutorials/send-emails-with-resend/index.md]:
```suggestion
Next, you’re presented with a list of DNS records to add to your Cloudflare domain. On your Cloudflare dashboard, select the domain you entered earlier and navigate to `DNS` > `Records`. Copy/paste the DNS records (DKIM, SPF, and DMARC records) from Resend to your Cloudflare domain.
```

File: content/workers/tutorials/send-emails-with-resend/index.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+![Image of adding DNS records to a Cloudflare domain](/images/workers/tutorials/resend/add_dns_records.png)
```
Comment (kodster28) [inline content/workers/tutorials/send-emails-with-resend/index.md:72]:
```suggestion
![Image of adding DNS records to a Cloudflare domain](/images/workers/tutorials/resend/add_dns_records.png)

{{<Aside type="note">}}

If you need more help adding DNS records in Cloudflare, refer to [Manage DNS records](/dns/manage-dns-records/how-to/create-dns-records/).
```

File: content/workers/tutorials/send-emails-with-resend/index.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+---
+$ npm start
+```
+
+And finally, deploy your Worker with `npm run deploy`.
+
+## Related resources
+
+- [Storing API keys and tokens with Secrets](/workers/configuration/secrets/).
+- [Transferring your domain to Cloudflare](/registrar/get-started/transfer-domain-to-cloudflare/).
```
Comment (kodster28) [inline content/workers/tutorials/send-emails-with-resend/index.md:183]:
```suggestion
- [Transferring your domain to Cloudflare](/registrar/get-started/transfer-domain-to-cloudflare/).
- [Send emails from Workers](/email-routing/email-workers/send-email-workers/)
```

**Review by kodster28 — APPROVED** [review]
approved from a content perspective

File: content/workers/tutorials/send-emails-with-resend/index.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+};
+```
+
+To test your code locally, run the following command and navigate to [http://localhost:8787/](http://localhost:8787/) in a browser:
+
+```bash
+---
+header: Local development server
+---
+$ npm start
```
Comment (irvinebroque) [inline content/workers/tutorials/send-emails-with-resend/index.md:127]: should this be `npx wrangler@latest dev` directly to help familiarize people with Wrangler CLI?

**Review by irvinebroque — APPROVED** [review]
Nice one! Should this page:

https://developers.cloudflare.com/email-routing/email-workers/send-email-workers/

Link to new template?

File: content/workers/tutorials/send-emails-with-resend/index.md (hunk 4 above)
Comment (megaconfidence) [reply content/workers/tutorials/send-emails-with-resend/index.md:127]: True, but that's way more typing to do

**Pull request merged**